packages/sirius-web/frontend/sirius-web-application/src/views/edit-project/workbench-views/useDetailsViewSubscription.tsx

```diff
@@ -60,21 +61,20 @@ export const useDetailsViewSubscription = (
 
   const variables: GQLDetailsEventVariables = { input };
 
+  const onComplete = () => setState((prevState) => ({ ...prevState, complete: true }));
+
   const onData = ({ data }: OnDataOptions<GQLDetailsEventSubscription>) => {
     const { data: gqlDetailsEventSubscription } = data;
     if (gqlDetailsEventSubscription) {
       const { detailsEvent: payload } = gqlDetailsEventSubscription;
       setState((prevState) => ({ ...prevState, payload, complete: false }));
       if (isFormRefreshedEventPayload(payload)) {
-        const { form } = payload;
-        setState((prevState) => ({ ...prevState, form, complete: false }));
+        setState((prevState) => ({ ...prevState, complete: false }));
```

Comment:
Same comment as the previous hook, you don't need any of those "if" to perform the `setState`